libs/platform/src/lib/approval-flow/approval-flow-add-node/approval-flow-add-node.component.ts

```diff
@@ -172,15 +164,14 @@ export class ApprovalFlowAddNodeComponent implements OnInit, OnDestroy {
 
     /** @hidden */
     ngOnInit(): void {
-        this._dataSourceSub = forkJoin([
-            this._data.approvalFlowDataSource.fetchTeams().pipe(take(1)),
-            this._data.approvalFlowDataSource.fetchApprovers().pipe(take(1))
-        ]).subscribe(([teams, approvers]) => {
+        this._data.teamsDataSource.match(new Map());
+        this._data.usersDataSource.match(new Map());
```

Comment:
Just a question, what does this .match do exactly.

Reply:
This is needed to trigger initial loading of data without any filters applied. I've made `match`'s function param optional, so it looks slightly better. Also added a comment explaining why this is needed